docs/using-draft-js.stories.mdx

```diff
@@ -0,0 +1,121 @@
+define an intitial `SelectionState` using this static method.
+
+#### Other useful methods
+* `EditorState.push(editorState: EditorState, contentState: ContentState, changeType: EditorChangeType)` - it is very
+unlikely that you will have a need to use this but if direct content changes are required they must be applied to the 
+`EditorState` using this method. It returns a new `EditorState` object with the specified `ContentState` applied as the 
+new currentContent, the `changeType` defines the operation being carried out on the state, see 
+https://draftjs.org/docs/api-reference-editor-change-type for the defined options.
+* The current `ContentState` can be accessed by calling the `getCurrentContent()` instance method.
+* The current `SelectionState` can be accessed by calling the `getSelection()` instance method.
+
+### ContentState
+`ContentState` is also an Immutable Record and represents the state of the editor's entire contents (text, block and 
+inline styles, and entity ranges) and its two selection states (before and after rendering). There are range of useful 
+methods accessible through `ContentState` that can provide information about the current content rendered in the editor, 
+for more dtsails it is best to refer to the API reference documentation 
+https://draftjs.org/docs/api-reference-content-state.
+
+#### Importing ContentState
+The `ContentState` can be imported either directly from `draft-js` or alternatively it has been exposed as part of the 
+`TextEditor` component's interface.
+
+```js
+import { ContentState } from 'draft-js';
+```
+```js
+import { TextEditorContentState } from 'carbon-react/lib/components/text-editor';
+```
+
+#### Useful static methods
+There are two statci methods that facilitate creating `ContentState`:
+* `createFromText(text: string, delimiter?: string)` - will generate state from a given string paramater, passing an 
+optional delimiter will define how the content blocks are split, if none isd provided the method will default to using 
+`\n`. This method will commonly be used with the `createWithContent` method surfaced by the `EditorState` like so:
+```js
+  const Foo = (props) => {
+    const [state, setState] = useState(EditorState.createWithContent(ContentState.createFromText('text content')));
+
+    return (
+      <Editor editorState={ state } onChange={ (newState) => setState(newState)} />
+    );
+  };
+```
+* `createFromBlockArray(blocks: Array<ContentBlock>, entityMap: ?OrderedMap)` - this method will produce a 
+`ContentState` object from an array of `ContentBlock`s, an optional Immutable map of `DraftEntity` records can also be 
+provided as the second argument. A `ContentBlock` is itself an Immutable Record that maintains the following 
+information about a block: a string key, the entity type, the block's text and an Immutable characterList which 
+maintains the styling and other data for each cahracter. Commonly this method will be used in conjunction with `Draft`'s 
```

Comment:
Typo on "character"

Reply:
Yeah I did a great job proof reading this 🤦

Reply:
Easily missed when you wrote it yourself!